### PR TITLE
add initial contact files

### PR DESCRIPTION
**Pull request by rcmadden**
Actually Ready for review.

### CODE REVIEW

File: neukinnis/client/contact/contact.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Neukinnis</title>
+    <link rel="stylesheet" type="text/css" href="style.css"
```
Comment (treyhunner) [inline neukinnis/client/contact/contact.html]: It looks like you're missing a `>`.

**Comment by treyhunner** [comment]
@rcmadden are you ready for code review on this?

**Comment by treyhunner** [comment]
When trying to run the tests I'm receiving failures for all of the tests.  3 of the 4 failures show:

    FAIL ReferenceError: Can't find variable: test

@rcmadden do the tests pass for you?  If so, is your local branch fully pushed to the server?

**Comment by jadedsurfer** [comment]
The html files should be directly in /client instead of /client/contact. The css files should be in /client/css. The tests should be in /spec/integration instead of in /client. 

Please resolve any conflicts with the css, gulpfile, and package.json by merging master into your branch. When you merge, it will say you have conflicts. You can resolve them through your IDE by going to VCS > git > Resolve Conflicts. If you need help with this, let us know.

**Comment by treyhunner** [comment]
I see two test failures currently:

```
FAIL contact us page title good
#    type: assertTitle
#    file: neukinnis/spec/integration/test-name.spec.js:16
#    code: test.assertTitle('Contact Us', 'contact us page title good');
#    subject: ""
#    expected: "Contact Us"
FAIL Cannot dispatch mousedown event on nonexistent selector: a[href="index.html"]
#    type: uncaughtError
#    file: neukinnis/spec/integration/test-name.spec.js:1355
#    error: Cannot dispatch mousedown event on nonexistent selector: a[href="index.html"]
#           CasperError: Cannot dispatch mousedown event on nonexistent selector: a[href="index.html"]
#               at mouseEvent (/usr/local/lib/node_modules/casperjs/modules/casper.js:1355)
#               at click (/usr/local/lib/node_modules/casperjs/modules/casper.js:462)
#               at neukinnis/spec/integration/test-name.spec.js:23
#               at runStep (/usr/local/lib/node_modules/casperjs/modules/casper.js:1553)
#               at checkStep (/usr/local/lib/node_modules/casperjs/modules/casper.js:399)
#    stack: not provided
```

**Pull request merged**